app/src/tun-local/kotlin/tunnel/Persistence.kt

```diff
@@ -93,24 +93,24 @@ class FiltersPersistence {
 }
 
 class RequestPersistence(
```

Comment:
Could you describe in a few sentences how the batch loading works in the new code? What's the algorithm?